### DISCUSSION TITLE
Does Sadtalker support the input of streaming audio or only wave files?

### DISCUSSION THREAD

**Discussion by maglore9900** [OpenTalker/SadTalker#954] [Q&A] [answered]
Does Sadtalker support the input of streaming audio or only wave files?

**Comment by samarasimhapeyala** [comment]
Sadtalker primarily supports WAV files for audio input. It doesn't natively support streaming audio input directly. If you want to use streaming audio, you would typically need to capture that audio, save it as a WAV file, and then process it with Sadtalker. Consider using libraries like `pydub` or `soundfile` to handle audio before passing it to Sadtalker.

**Reply by samarasimhapeyala** [reply, marked as answer]
@maglore9900 can you mark it as answered if found useful, as of now it's just closed.
